templates/products/ils.html

```diff
@@ -147,10 +147,10 @@ <h6>REST API</h6>
     <h6 class="section-subtitle">InvenioILS relies on Invenio Framework's core which uses some of the world's top open source products</h6>
     <div class="row">
       <div class="col-md-6">
-        <img class="flavour" src="{{ '/static/img/logo-elasticsearch.svg'|asseturl }}"/>
-        <h6>Elasticsearch</h6>
+        <img class="flavour" src="{{ '/static/img/logo-opensearch.svg'|asseturl }}"/>
```

Comment:
it looks like that adding to the [css file](https://github.com/inveniosoftware/inveniosoftware.org/blob/master/assets/static/css/style.css#L17):
```

img.flavour {
  max-width: 100%;
}
```
fixes the issue of having the SVG too large :)